build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 
 if [ ! -s "$TRAVIS_TAG" ] ; then
```

Comment:
This check doesn't even work in the original. I seem to always see the text:`"This build will be published under the tag:` when the variable isn't specified.